coalib/output/ConsoleInteraction.py

```diff
@@ -664,22 +667,22 @@ def choose_action(console_printer, actions, apply_single=False):
                 actions_desc_len = len(actions_desc)
                 if c.isnumeric():
                     for i, action in enumerate(actions, 0):
-                        c = int(c)
-                        if i == c:
+                        if i == int(c):
                             actions_desc.append(action.desc)
                             actions_name.append(action.name)
                             break
                 elif c.isalpha():
                     c = c.upper()
-                    c = '(' + c + ')'
+                    str_c = '(' + c + ')'
                     for i, action in enumerate(actions, 1):
-                        if c in action.desc:
+                        if str_c in action.desc:
                             actions_desc.append(action.desc)
                             actions_name.append(action.name)
                             break
                 if actions_desc_len == len(actions_desc):
-                    console_printer.print(format_lines(
-                        'Please enter a valid letter.', symbol='['))
+                    console_printer.print(WARNING_COLOR +
+                                          STR_INVALID_OPTION.format(c) +
```

Comment:
This is wrong. Look at how color is passed to `console_printer` in other places in the file.